schema/360-giving-schema.json

```diff
@@ -726,7 +726,7 @@
             "ZM",
             "ZW"
           ],
-          "description": "The ISO Country Code of the location of this activity.",
+          "description": "The 2-character ISO Country Code of the location of this activity. Taken from [ISO_3166-1_alpha-2](https://www.iso.org/iso-3166-country-codes.html)",
```

Comment:
I'm not sure why underscores are being used here - the name of the code is `ISO 3166-1 alpha-2`